GVRf/Framework/src/org/gearvrf/GVRAssimpImporter.java

```diff
@@ -31,7 +33,15 @@
     int getNumberOfMeshes() {
         return NativeAssimpImporter.getNumberOfMeshes(getPtr());
     }
-
+    
+    /**
+     * @return Loads the 3D model.
```

Comment:
JavaDoc on internal methods is a lot less critical than JavaDoc on public methods ... but this isn't very well-formed: "Loads the 3D model." makes sense as a method summary, not an `@return`, and it probably would be a very good idea to document the `@param bitmap`.